src/instruction.ts

```diff
@@ -40,10 +40,8 @@ export function avrInstruction(cpu: ICPU) {
     sreg |= sum & 256 ? 1 : 0;
     sreg |= 1 & ((d & r) | (r & ~R) | (~R & d)) ? 0x20 : 0;
     cpu.data[95] = sreg;
-  }
-
-  /* ADD, 0000 11rd dddd rrrr */
```

Comment:
Let's also move the first comment, `  /* ADC, 0001 11rd dddd rrrr */` to live inside the if statement, to keep it consistent with the rest

Reply:
Done